skycultures/Book_of_Fixed_Stars_English/info.ini

```diff
@@ -0,0 +1,7 @@
+﻿[info]
+name = Book of Fixed Stars

```

Comment:
I think it would be better to have a shorter name, else it will be cut off in the GUI table. (Like "Book of Fi...")
"Sufi" or "Al-Sufi" should be enough. 
Also, the directory name should be shorter and use small letters only. I propose "sufi".

Reply:
done
sky culture is now "Al-Sufi" and directory is now "al-sufi".